<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>344</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ATUALIZAÇÃO DE COLABORADOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Back_Editar.jpg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background: transparent</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent</string>
            </property>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>61</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0,.7);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-radius: 5px</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;NOME&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ln_nome">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
padding-left: 10px</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_13" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent</string>
            </property>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0,.7);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-radius: 5px</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;FUNÇÃO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ln_funcao">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
padding-left: 10px</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_14" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent</string>
            </property>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>101</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0,.7);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-radius: 5px</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ADMISSÃO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ln_admissao">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
padding-left: 10px</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="styleSheet">
          <string notr="true">background: transparent</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent</string>
            </property>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0,.7);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-radius: 5pX</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARGO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ln_cargo">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
padding-left: 10px</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0,.7);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-radius: 5px</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;HORÁRIO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ln_horario">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
padding-left: 10px</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>92</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="bt_salvar">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(0, 0, 255);
	font: 75 18pt &quot;Arial&quot;;
	background-color: rgb(255, 255, 0);
	border-style: outset;
	border-width: 1px;
	border-color: blue;
	border-radius: 5px
}
QPushButton:pressed{
	background-color: rgb(0, 0, 255);
	color: rgb(255, 255, 0);
	border-color:yellow;
}</string>
            </property>
            <property name="text">
             <string>SALVAR</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Back_Editar.qrc"/>
 </resources>
 <connections/>
</ui>
